pyproject.toml

```diff
@@ -30,6 +30,7 @@ dependencies = [
   "grpcio==1.70.0",
   "multidict<7.0,>=4.5",
   "msgpack>=1,<1.2",
+  "otaclient_iot_logging_server_pb2 @ https://github.com/tier4/otaclient-iot-logging-server/releases/download/v1.3.0/otaclient_iot_logging_server_pb2-1.0.0-py3-none-any.whl#sha256:e5a2e6474a8b6f5656679877a5df40891e6a65c29830a31faed3d47d9973be60",
```

Comment:
So far, otaclient-iot-logging-server package is not released into PyPI yet. This will be future task.

Reply:
Good point.
As long as it is for internal use only, I think we don't have to publish.
In future, if ota-client and it's logger is used by someone outside of TIER IV, I think it's better to register into PyPI as open source project.

Reply:
( Oh, that would be a very very future topic. I got your point :smile: .